index.bs

```diff
@@ -1196,7 +1196,11 @@ const inputBuffer1 = new Float32Array(TENSOR_SIZE).fill(1);
 const inputBuffer2 = new Float32Array(TENSOR_SIZE).fill(1);
 
 // Asynchronously execute the compiled model with the specified inputs.
-let outputs = await compilation.compute({'input1': {buffer: inputBuffer1}, 'input2': {buffer: inputBuffer2}});
```

Comment:
eslint error: exceeds max-len (80).

Reply:
The related eslint rule is [max-len](https://eslint.org/docs/rules/max-len#enforce-a-maximum-line-length-max-len) whose default value is 80.